.github/workflows/post_release.yaml

```diff
@@ -171,16 +168,17 @@ jobs:
     runs-on: ${{ matrix.os.image }}
 
     steps:
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+      
       - uses: actions/checkout@v2
 
       - name: Install released version of codegen
         run: cargo install flutter_rust_bridge_codegen
 
       - uses: dart-lang/setup-dart@v1
```

Comment:
(save as above)

Reply:
done in c8575ce64065d66ed7ae9ef2f171d322c5a02a9b